job_bundles/README.md

```diff
@@ -0,0 +1,277 @@
+`objectType: DIRECTORY` and `dataFlow: OUT`:
+
+```yaml
+- name: OutputDir
+  type: PATH
+  objectType: DIRECTORY
+  dataFlow: OUT
+  userInterface:
+    control: CHOOSE_DIRECTORY
+    label: Output Directory
+    groupLabel: Render Parameters
+  default: "./output"
+  description: Choose the render output directory.
+```
+
+The value of `OutputDir` is treated by Job Attachments as a directory where the Job is expected to write output files.
+
+The remainder of the `blender_render` sample's Job Template defines the workflow as a single Step with each frame of the animation
+being rendered as a separate Task:
+
+```yaml
+steps:
+- name: RenderBlender
+  parameterSpace:
+    taskParameterDefinitions:
+    - name: Frame
+      type: INT
+      range: "{{Param.Frames}}"
+  script:
+    actions:
+      onRun:
+        command: bash
+        # Note: {{Task.File.Run}} is a variable that expands to the filename on the Worker host's
+        # disk where the contents of the 'Run' embedded file, below, is written.
+        args: ['{{Task.File.Run}}']
+    embeddedFiles:
+      - name: Run
+        type: TEXT
+        data: |
+          # Configure the task to fail if any individual command fails.
+          set -xeuo pipefail
+
+          mkdir -p '{{Param.OutputDir}}'
+
+          blender --background '{{Param.BlenderSceneFile}}' \
+                  --render-output '{{Param.OutputDir}}/{{Param.OutputPattern}}' \
+                  --render-format {{Param.Format}} \
+                  --use-extension 1 \
+                  --render-frame {{Task.Param.Frame}}
+```
+
+Additional samples of Job Templates can be found in all of the Job Bundles in this repository as well as in the 
+[samples provided](https://github.com/OpenJobDescription/openjd-specifications/tree/mainline/samples)
+by Open Job Description. Please see https://github.com/OpenJobDescription/openjd-specifications/wiki
+for more information on Job Templates.
+
+### Elements - Parameter Values
+
+The `parameter_values.json`/`parameter_values.yaml` file in a Job Bundle gives you a place to "bake"
+the values of some of the Job Parameters, and/or [deadline:CreateJob API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateJob.html)
+request arguments into the Job Bundle so that the values do not have to be given when submitting a Job using the
+Job Bundle. The Job Bundle submission UI that is created will still allow values for these parameters to be input.
+
+The format of the file, in YAML, is:
+
+```yaml
+parameterValues:
+- name: <string>
+  value: <integer>, <float>, or <string>
+- name: <string>
+  value: <integer, <float>, or <string>
+... repeating as necessary
+```
+
+Each element of the `parameterValues` list in the file must be either: 
+
+1. A Job Parameter defined in the Job Bundle's Job Template; 
+2. A Job Parameter defined in a Queue Environment on the Queue that you are submitting the Job to; or 
```

Comment:
I didn't know this! Perhaps we find a way to link out to one of the Queue Environment samples.

```diff
-A Job Parameter defined in a Queue Environment on the Queue that you are submitting the Job to; or 
+A Job Parameter defined in a Queue Environment on the Queue that you are submitting the Job to;
```